<ui version="4.0" >
 <class>SIPAccountSettings</class>
 <widget class="QWidget" name="SIPAccountSettings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>SIP Account Settings</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="4" column="0" colspan="4" >
    <widget class="QCheckBox" name="forceProxy" >
     <property name="contextMenuPolicy" >
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="layoutDirection" >
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text" >
      <string>Force register through proxy</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4" >
    <widget class="QCheckBox" name="activateSIMPLE" >
     <property name="layoutDirection" >
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text" >
      <string>Activate presence and chat (SIP/SIMPLE)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" >
    <widget class="QLabel" name="label_proxyport" >
     <property name="text" >
      <string>:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3" >
    <widget class="QLineEdit" name="realmLineEdit" />
   </item>
   <item row="1" column="1" colspan="3" >
    <widget class="QLineEdit" name="displaynameLineEdit" />
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_proxy" >
     <property name="text" >
      <string>Proxy:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QLineEdit" name="proxyServerLineEdit" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>SIP Domain / Realm:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" >
    <widget class="QLineEdit" name="proxyPortLineEdit" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maxLength" >
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3" >
    <widget class="QLineEdit" name="registerServerLineEdit" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_server" >
     <property name="text" >
      <string>Server:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>Displayname:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>realmLineEdit</tabstop>
  <tabstop>displaynameLineEdit</tabstop>
  <tabstop>registerServerLineEdit</tabstop>
  <tabstop>proxyServerLineEdit</tabstop>
  <tabstop>proxyPortLineEdit</tabstop>
  <tabstop>forceProxy</tabstop>
  <tabstop>activateSIMPLE</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
